### ISSUE TITLE
Pre-commit checks are failing

### ISSUE THREAD

**Issue by amanmdesai** [hsf-training/hsf-training-cpp-webpage#22]

**Comment by amanmdesai** [comment]
see for instance #21

**Comment by klieret** [comment]
Right. This is because I've enabled the bot but the config is missing. I thought I had automatically added that to every repository, but somehow that apparently didn't work. Strange.

**Comment by klieret** [comment]
@amanmdesai Btw, it's not clear if this training module and the "intermediate" one are still in active development. There is consistent effort on https://github.com/hsf-training/cpluspluscourse however

**Comment by amanmdesai** [comment]
> Right. This is because I've enabled the bot but the config is missing. I thought I had automatically added that to every repository, but somehow that apparently didn't work. Strange.

I see. Thanks.

**Comment by amanmdesai** [comment]
> @amanmdesai Btw, it's not clear if this training module and the "intermediate" one are still in active development. There is consistent effort on https://github.com/hsf-training/cpluspluscourse however

Oh okay. Thanks for the link.

**Comment by amanmdesai** [comment]
> Right. This is because I've enabled the bot but the config is missing. I thought I had automatically added that to every repository, but somehow that apparently didn't work. Strange.

Hi @klieret, I was just curious to learn how the bot works and how its set-up. Could you please point me to a reference/code on how this can be done on a repo?

Thanks.

**Comment by klieret** [comment]
Hi @amanmdesai : https://pre-commit.com/ is a good resource for this. This tells you how to set it up in your repository and your local git. https://pre-commit.ci/ is then the version that then automatically also runs these tests as a github CI system

**Comment by amanmdesai** [comment]
Thanks very much for the references.
